content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -1,130 +1,106 @@
+You must define input variables manually; Terraform Cloud does not automatically discover variable names from Terraform code. However, workspaces that use Terraform 0.10.0 or later allow you to commit any number of [`*.auto.tfvars` files](/docs/language/values/variables.html#variable-files) to provide default Terraform input variables. If any variables from these files have the same key as existing variables in the Terraform Cloud workspace, the variables applied to the workspace overwrite variables from the files.
 
-To delete a variable, click its "🗑" (trash can) icon, then confirm your decision in the dialog box that appears.
+### Environment Variables
 
-You can edit one variable at a time, and must save or cancel your current edits before editing other variables in the list.
+Terraform Cloud performs Terraform runs on disposable Linux worker VMs using a POSIX-compatible shell. Before running Terraform operations, Terraform Cloud uses the `export` command to populate the shell with environment variables.
 
-![Screenshot: A variable being edited](./images/vars-edit.png)
+Environment variables can store provider credentials and other data. Refer to your provider's Terraform Registry documentation for a full list of supported shell environment variables (e.g., authentication variables for [AWS](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#environment-variables), [Google Cloud Platform](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/getting_started#adding-credentials), and [Azure](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#argument-reference)). Environment variables can also [modify Terraform's behavior](/docs/cli/config/environment-variables.html). For example, `TF_LOG` enables detailed logs for debugging.
 
-### Multi-line Values
+#### Parallelism
 
-The text fields for variable values can handle multi-line text (typed or pasted) without any special effort.
+You can use the `TFE_PARALLELISM` environment variable when your infrastructure providers produce errors on concurrent operations or use non-standard rate limiting. The `TFE_PARALLELISM` variable sets the  `-parallelism=<N>` flag for  `terraform plan` and `terraform apply`  ([more about `parallelism`](/docs/internals/graph.html#walking-the-graph)). Valid values are between 1 and 256, inclusive, and the default is `10`. Terraform Cloud Agents do not support `TFE_PARALLELISM`, but you can specify flags as environment variables directly via [TF_CLI_ARGS](/docs/cli/config/environment-variables.html#tf-cli-args). In these cases, use `TF_CLI_ARGS="parallelism=<N>"` instead.
 
-### HCL Values
+~> **Warning:** We recommend talking to HashiCorp support before setting `TFE_PARALLELISM`.
 
-Variable values are strings by default. To enter list or map values, click the variable's "HCL" checkbox (visible when editing) and enter the value with the same HCL syntax you would use when writing Terraform code. For example:
+## Scope
```

Comment:
I feel like this section is kind of a repeat of what we talked about in the page intro. It might be nicer to provide a reference table here with ways to set variables and their scopes, so that people can quickly find the scope they want the variable to have and match it to a way to set the variable?

Reply:
Judith - this is an amazing idea. I'm going to implement it now.